openaerostruct/mphys/aero_builder.py

```diff
@@ -0,0 +1,140 @@
+"""
+Class definition for the Mphys builder for the aero solver.
+"""
+
+import copy
+
+import openmdao.api as om
+
+from openaerostruct.mphys.utils import get_number_of_nodes
+from openaerostruct.mphys.aero_mesh import AeroMesh
+from openaerostruct.mphys.demux_surface_mesh import DemuxSurfaceMesh
+from openaerostruct.mphys.mux_surface_forces import MuxSurfaceForces
+from openaerostruct.mphys.aero_solver_group import AeroSolverGroup
+from openaerostruct.mphys.aero_funcs_group import AeroFuncsGroup
+
+try:
+    from mphys.builder import Builder
+    from mphys.distributed_converter import DistributedConverter, DistributedVariableDescription
+except ImportError:
+    pass
```

Comment:
Instead of passing this error, can you set a flag to identify if mphys is installed, and then throw the actual error if someone tries to initialize code that requires mphys?

Here is an example of what we do for [pyGeo].(https://github.com/mdolab/pygeo/blob/0e6b35609cd79eaa9c3f163ef90f0cf20352c7b1/pygeo/parameterization/DVGeoVSP.py#L15-L93)